govuk-paas/paas-cdn-broker
# [#164986888] First round of changes: extra logging, move to go mod, and copy in the lego/acme package

## What

This PR includes a few things:

* Logging improvements brought in from upstream;
* Move Go packaging from glide to `go mod`;
* Copy the `github.com/xenolf/lego` package into this project and remove lots of its code.

Clearly these aren't minor changes, but a lot of it is things upstream already did or moving files around.

## How to review

* See if the testsuite passes;
* Deploy to a dev env and check you can provision/update/deprovision a CDN instance.

We think deploying this is wise until we have more confidence around the CDN Broker. Hopefully followup PRs will help with that.

## Who can review

Not @46bit @LeePorte
